src/1_utils.js

```diff
@@ -1335,49 +1335,39 @@ utils.shouldAddDMAParams = function(endPointURL) {
 };
 
 utils.setDMAParams = function(data, dmaObj = {}, endPoint) {
-	const DMA_EEA = "dma_eea";
-	const DMA_Ad_Personalization = "dma_ad_personalization";
-	const DMA_Ad_User_Data = "dma_ad_user_data";
-
+	const v1_DMAEndPoints = [ "/v1/open", "/v1/pageview" ];
+	const v2_DMAEndPoints = [ "/v2/event/standard", "/v2/event/custom" ];
 	const dmaParams = {
-		[DMA_EEA]: dmaObj.eeaRegion || false,
-		[DMA_Ad_Personalization]: dmaObj.adPersonalizationConsent || false,
-		[DMA_Ad_User_Data]: dmaObj.adUserDataUsageConsent || false
+		dma_eea: dmaObj['eeaRegion'],
```

Comment:
Let's consolidate our parameter names as well

Reply:
🤦 Native SDK didn't consolidated these params?
Alright. let's leave things as it is.